scylla/src/utils/pretty.rs

```diff
@@ -42,7 +42,10 @@ where
             CqlValue::Text(t) => write!(f, "{}", CqlStringLiteralDisplayer(t))?,
             CqlValue::Blob(b) => write!(f, "0x{:x}", HexBytes(b))?,
             CqlValue::Empty => write!(f, "0x")?,
-            CqlValue::Decimal(d) => write!(f, "{}", d)?,
+            CqlValue::Decimal(d) => {
+                let (bytes, scale) = d.as_signed_be_bytes_slice_and_exponent();
+                write!(f, "blobAsDecimal(0x{:04x}{:x})", scale, HexBytes(bytes))?
```

Comment:
`scale` should be printed as four bytes, not four digits - there should be 8 digits in total.

Besides, the protocol spec requires the value to be serialized in big-endian. The way you print the value conforms to that, but it's a little non-obvious. Perhaps you could use [to_be_bytes](https://doc.rust-lang.org/std/primitive.i32.html#method.to_be_bytes) here?